core/meta/src/TViewPubDataMembers.h

```diff
@@ -75,9 +75,6 @@ class TViewPubDataMembers : public TList {
    void       RecursiveRemove(TObject *obj) override;
    TObject   *Remove(TObject *obj) override;
    TObject   *Remove(TObjLink *lnk) override;
-
-public:
-   ClassDefInlineOverride(TViewPubDataMembers, 0)
```

Comment:
A `ClassDef` macro is required for any classes that derives from a class with its own `ClassDef`.   In particular this is the cases for any class that derives directly or indirectly from `TObject`.

Without the `ClassDef` macro, several virtual function used to identify the actual type of a `TObject`.

In this case, this means that if a routine  (eg. `clingwrapper`) ask about of an object of type `TViewPubDataMembers`, it will get inaccurate information and thus proceed with erroneous behavior.